src/components/templates/Audience/Audience.tsx

```diff
@@ -143,6 +144,7 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
   const baseRows = venue?.auditoriumRows ?? DEFAULT_AUDIENCE_ROWS_NUMBER;
 
   const { isShown: isUserAudioOn, toggle: toggleUserAudio } = useShowHide(true);
+  const { isLoaded: settingsAreLoaded, settings } = useSettings();
```

Comment:
`useSettings` is setting a global state in the `customizations` document which will be used for all venues on the platform. Is this behaviour expected?

Reply:
`settingsAreLoaded` -> `areSettingsLoaded`

Reply:
@denisdimitrov, yes. Our aim is to be able to hide the features that can be a bottleneck for us in terms of performance. It's done this way to be able to turn off the reactions in firebase for everyone.

Reply:
fixed in 2bf823d11ad27c3992680c9ac6a4585293744884